### PR TITLE
Use dbus instead of internal service

### LINKED ISSUE #3: Watch for new outputs
The daemon only sets the color for outputs that were presented on the startup: if I start the daemon, then connect a new display and then try to modify the color, newly connected display is unaffected.

### PR DESCRIPTION
**Pull request by jeremija**
The same interface is implemented as
[`wl-gammarelay-rs`](https://github.com/MaxVerevkin/wl-gammarelay-rs).

Usage examples:

    busctl --user -- call rs.wl-gammarelay / rs.wl.gammarelay UpdateTemperature n -500
    busctl --user -- call rs.wl-gammarelay / rs.wl.gammarelay UpdateTemperature n +500

    busctl --user -- call rs.wl-gammarelay / rs.wl.gammarelay UpdateBrightness d -0.2
    busctl --user -- call rs.wl-gammarelay / rs.wl.gammarelay UpdateBrightness d +0.2

    busctl --user -- set-property rs.wl-gammarelay / rs.wl.gammarelay Brightness d 0.5
    busctl --user -- set-property rs.wl-gammarelay / rs.wl.gammarelay Brightness d 1

    busctl --user -- set-property rs.wl-gammarelay / rs.wl.gammarelay Temperature q 4000
    busctl --user -- set-property rs.wl-gammarelay / rs.wl.gammarelay Temperature q 6500

Introspection:

    $ busctl --user introspect rs.wl-gammarelay / rs.wl.gammarelay
    NAME               TYPE      SIGNATURE RESULT/VALUE FLAGS
    .UpdateBrightness  method    d         -            -
    .UpdateTemperature method    n         -            -
    .Brightness        property  d         1            emits-change writable
    .Temperature       property  q         6500         emits-change writable

Closes #3

### CODE REVIEW

**Comment by jeremija** [comment]
cc @MaxVerevkin I implemented the same DBUS interface as you :slightly_smiling_face:

**Comment by MaxVerevkin** [comment]
Nice! Now `i3status-rust` can work with both projects using the same driver.

File: dbus.go [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+	var value float64
+
+	switch t := v.Value().(type) {
+	case float64:
+		value = t
+	case *float64:
+		value = *t
+	default:
+		return dbus.MakeFailedError(fmt.Errorf("value is not int16: %T", v.Value()))
```
Comment (MaxVerevkin) [inline dbus.go]: Copy-paste error in the message :)

Comment (jeremija) [reply dbus.go]: Thanks, fixed!

**Pull request merged**